### PR TITLE
[OY-19563] Issues with Custom text to validation fields

### PR DESCRIPTION
**Pull request by charmcitygavin**
## Purpose

Some measures did not have the correct custom label set for totals in the `DeviationFromMeasureSpec` component. I kept the method for retrieving the custom label consistent with what was already there by accessing the correct custom label from `PMD.qualifiers`.

#### Linked Issues to Close

Closes [OY-19563](https://qmacbis.atlassian.net/browse/OY2-19563).

## Approach

The custom total label can exist in the `PMD.qualifiers` array from a measure's corresponding `data.ts` file. Not all measures have a custom total label, but those that do include it as the last item in this array.

One of the existing measures accesses this label using this method: `PMD.qualifiers.slice(-1)[0]`, which I have left in place because it works fine. However, other measures accessed the item directly, i.e. `PMD.qualifiers[3]`. Because more measures used this method than the first method, this is the method I chose for measures that were missing this label altogether in order to keep things as consistent as possible.

Adding this label to `DeviationFromMeasureSpec` also adds it to corresponding validation warnings.

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/measures/2021/DEVCH/index.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const DEVCH = ({
               <CMQ.PerformanceMeasure data={PMD.data} calcTotal hybridMeasure />
               <CMQ.DeviationFromMeasureSpec
                 categories={PMD.categories}
-                customTotalLabel="Children"
+                customTotalLabel={PMD.qualifiers[3]}
```
Comment (jphechter) [inline services/ui-src/src/measures/2021/DEVCH/index.tsx:51]: As far as I can tell, this is the only place in the app where `customeTotalLabel` was actually different than the last qualifier (albeit that difference was "Children" and not "Children Total").

Considering that in every measure we're making the total field the label used for the last qualifier, can we just drop this logic altogether?

**Review by jphechter — COMMENTED** [review]
What's here definitely works and follows the established pattern. If we can simplify this component and kill `customTotalLabel` I'd be for it.

File: services/ui-src/src/measures/2021/CDFHH/index.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ export const CDFHH = ({
                 calcTotal
               />
               <CMQ.DeviationFromMeasureSpec
-                customTotalLabel="Total (Age 12 and older)"
```
Comment (benjaminpaige) [inline services/ui-src/src/measures/2021/CDFHH/index.tsx:54]: thank you for changing these CDF measures

**Review by benjaminpaige — APPROVED** [review]

**Comment by benjaminpaige** [comment]
looks good, just a couple tests need to be fixed.

File: services/ui-src/src/measures/2021/DEVCH/index.tsx (hunk 1 above)
Comment (charmcitygavin) [reply services/ui-src/src/measures/2021/DEVCH/index.tsx:51]: Great call! I hadn't realized that the DeviationFromMeasureSpecification already sets the total label based on the item with the `isTotal` flag. I'll refactor this logic out 👍🏻

File: services/ui-src/src/measures/2021/CommonQuestions/DeviationFromMeasureSpecification/index.tsx [hunk 3]
```diff
@@ -140,15 +136,6 @@ export const DeviationFromMeasureSpec = ({
...
         }
-        // A total category should have the label "Total", per the Figma design.
-        const totalIndex = rates.singleCategory.findIndex(
-          (cat: any) => cat.isTotal === true
-        );
-        if (totalIndex >= 0) {
-          rates.singleCategory[totalIndex].label = `${
-            customTotalLabel ? `${customTotalLabel} ` : "Total"
-          }`;
-        }
```
Comment (jphechter) [inline services/ui-src/src/measures/2021/CommonQuestions/DeviationFromMeasureSpecification/index.tsx:151]: 🙌 🤝 🤜 🤛 🚀 🌕

File: services/ui-src/src/measures/2022/CommonQuestions/DeviationFromMeasureSpecification/index.tsx [hunk 4]
```diff
@@ -141,15 +137,6 @@ export const DeviationFromMeasureSpec = ({
...
         }
-        // A total category should have the label "Total", per the Figma design.
-        const totalIndex = rates.singleCategory.findIndex(
-          (cat: any) => cat.isTotal === true
-        );
-        if (totalIndex >= 0) {
-          rates.singleCategory[totalIndex].label = `${
-            customTotalLabel ? `${customTotalLabel} ` : "Total"
-          }`;
-        }
```
Comment (jphechter) [inline services/ui-src/src/measures/2022/CommonQuestions/DeviationFromMeasureSpecification/index.tsx:152]: same as above

**Review by jphechter — APPROVED** [review]
-142 lines

LET'S GOOOO!!!!

**Pull request closed without merging**